packages/server/lib/util/ci_provider.js

```diff
@@ -445,8 +446,8 @@ const _providerCommitParams = () => {
       message: env.CI_COMMIT_MESSAGE,
       authorName: env.GITLAB_USER_NAME,
       authorEmail: env.GITLAB_USER_EMAIL,
-      // remoteOrigin: ???
-      // defaultBranch: ???
+      remoteOrigin: env.CI_PROJECT_URL,
```

Comment:
The `remoteOrigin` should be the url of the Git repo, not Gitlab's CI url. The `remoteOrigin` value should actually be mapped to `CI_REPOSITORY_URL`

Reply:
You're right, fixed it, thanks!